include/greentea-client/test_env.h

```diff
@@ -110,8 +230,48 @@ extern "C" {
 /**
  *  Greentea-client C API
  */
+
+/**
+ * Handshake with the host and send setup data (timeout and host test name)
+ *
+ * @details This function will send a preamble to the host.
```

Comment:
This needs the same grammatical corrections I suggested above, as it's a copy-paste.